mta-sts.md

```diff
@@ -617,6 +600,33 @@ A suggested workflow to implement such an opt out is as follows:
    allowing overlapping policy caches--safely remove the TXT record and HTTPS
    endpoint.
 
+## Preserving MX Candidate Traversal
+
+Implementors of send-time MTA-STS validation in mail transfer agents should take
+note of the risks of modifying the logic of traversing MX candidate lists.
+Because an MTA-STS policy can be used to prefilter invalid MX candidates from
+the MX candidate list, it is tempting to implement a "two-pass" model, where MX
+candidates are first filtered for possible validity according to the MTA-STS
+policy, and then the remaining candidates attempted in order as without an
+MTA-STS policy.  This may lead to incorrect implementations, such a message
+loops; implementors are instead recommended to traverse the MX candidate list as
+usual, and treat invalid candidates as though they were unreachable (i.e., as
+though there were some transient error when trying to deliver to that
+candidate).
+
+One consequence of maintaining MX candidate traversal order is that `report`-mode
+policies may show failures even if a message would, in `enforce`-mode,
```

Comment:
The policy fails for the high-priority MX host in either case. And the mail is delivered in either case, be it to different MX hosts.  There should a "tlsrpt" for the failing high-priority host in either case!

Reply:
So this entire paragraph goes.

Reply:
I guess. I wanted to call out that things that manifest as reported errors don't necessarily mean the message would not deliver, but that may be unnecessary to call out.